src/scss/utilities/variables/_colors.scss

```diff
@@ -5,3 +5,10 @@
 
 // define colors here in the format of:
 //$abbreviation-color-pink: #ff0099; // like $saa-color-pink
+$st-color-fog-light: #f4f4f4;
+$st-color-black: #2e2d29;
+$st-color-cardinal-red: #8c1515;
+$st-color-sky-blue: #016895;
+$st-color-brick: #7f2d48;
+$st-color-palo-verde: #017e7c;
+$st-color-outline-blue: #0098DB;
```

Comment:
I believe most if not all of these color variables are in Decanter already. Please take a look at:
https://github.com/SU-SWS/decanter/blob/master/core/src/scss/utilities/variables/core/_colors.scss